OuterWildsRandomSpeedrun/OuterWildsRandomSpeedrun.cs

```diff
@@ -20,8 +20,12 @@ public class OuterWildsRandomSpeedrun : ModBehaviour
         private string _goalPointName;
         private IModButton _speedrunButton; 
         private DateTime _startTime;
+        private DateTime _endTime;
         private ScreenPrompt _timerPrompt;
         private bool _modEnabled = false;
+        private Mesh marshmallowMesh;
```

Comment:
prefix with _ if we're following that standard

Reply:
done